web/template/watch.gohtml

```diff
@@ -13,297 +13,345 @@
-                            for {{.LectureHallName}}</h6>
-                        <div class="w-full overflow-x-scroll scrollbarThin">
-                            <div class="flex flex-row gap-x-2 pb-4 pl-2">
-                                {{range $preset := .Presets}}
-                                    <div style="min-width: 150px" class="text-center relative group">
-                                        <img id="presetImage{{$preset.LectureHallId}}-{{$preset.PresetID}}"
-                                             src="/public/{{if $preset.Image}}{{$preset.Image}}{{else}}noPreset.jpg{{end}}"
-                                             alt="prev" width="150px"
-                                             class="border-1 dark:border-slate-400 border-slate-500 rounded-lg">
-                                        <i onclick="watch.usePreset({{$course.Model.ID}}, {{$preset.LectureHallId}}, {{$preset.PresetID}})"
-                                           title="Switch to this preset"
-                                           class="opacity-0 group-hover:opacity-100 absolute top-1 right-1 p-1 rounded text-white bg-info fas fa-exchange-alt cursor-pointer"></i>
-                                    </div>
-                                {{end}}
-                            </div>
+            </div>
+            {{if .Presets}}
+                <div class="flex">
+                    <i class="fa-solid fa-camera mr-1 text-3 mt-1"></i>
+                    <div class="w-full overflow-x-scroll scrollbarThin">
+                        <div class="flex flex-row gap-x-2 pb-4 pl-2">
+                            {{range $preset := .Presets}}
+                                <div style="min-width: 150px" class="text-center relative group">
+                                    <img id="presetImage{{$preset.LectureHallId}}-{{$preset.PresetID}}"
+                                         src="/public/{{if $preset.Image}}{{$preset.Image}}{{else}}noPreset.jpg{{end}}"
+                                         alt="prev" width="150px"
+                                         class="border-1 dark:border-slate-400 border-slate-500 rounded-lg">
+                                    <i onclick="watch.usePreset({{$course.Model.ID}}, {{$preset.LectureHallId}}, {{$preset.PresetID}})"
+                                       title="Switch to this preset"
+                                       class="opacity-0 group-hover:opacity-100 absolute top-1 right-1 p-1 rounded text-white bg-info fas fa-exchange-alt cursor-pointer"></i>
+                                </div>
+                            {{end}}
                         </div>
                     </div>
-                {{end}}
+                </div>
             {{end}}
-            <!-- Video Sections -->
-            {{if $stream.VideoSections}}
+        {{end}}
+
+        <!-- Video Sections -->
+        {{if $stream.VideoSections}}
+            <div class="basis-full p-5 lg:order-none order-5">
```

Comment:
I think it would look better to go back to a design similar to the beginning where everything is left aligned.

![grafik](https://user-images.githubusercontent.com/30267166/192628708-b2dc92f4-146e-4489-ad75-c4cc6ce1ba8b.png)


Maybe we can also a separator line above video sections, descriptions etc. I see this lines are used on YT and vimeo?

Reply:
Done. Added some separators too!